build/artifacts.props

```diff
@@ -31,6 +31,7 @@ This can be done once #4246 is complete, and done in conjunction with converting
     <PackageArtifact Include="Microsoft.AspNetCore.AzureAppServices.HostingStartup" Category="ship" />
     <PackageArtifact Include="Microsoft.AspNetCore.AzureAppServicesIntegration" Category="ship" />
     <PackageArtifact Include="Microsoft.AspNetCore.Blazor" Category="ship" />
+    <PackageArtifact Include="Microsoft.AspNetCore.Blazor.Build" Category="ship" />
```

Comment:
Does this need to be in ship or just the components part?

Reply:
We're shipping all the Blazor.* packages (to NuGet) as well as the Components.* ones (some of which will be in shared framework).